target/linux/mvebu/config-4.9

```diff
@@ -60,6 +60,8 @@ CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_MQ_PCI=y
 CONFIG_BOUNCE=y
+CONFIG_BTRFS_FS=y
```

Comment:
No, this shouldn't be enabled by default for mvebu. Use the package instead if you need support however I don't see why this should be in by default.